README.md

```diff
@@ -36,8 +36,7 @@ Algernon allows for keyboard only interaction - no need to use your mouse!
 - Search global variables
 - Print and evaluate Smalltalk expressions 
 - Action verbs for filtering and quick access (e.g. `run SomeTest` will only search for tests and set a default action)
-- Full text search in classes
-- Save and/or quit the image
```

Comment:
Why exactly did you remove this? Or is this a merge thing?

In any case: Please change it.

Reply:
This is a Merge Error; I restored it.